scripts/install-dependencies.sh

```diff
@@ -45,27 +41,14 @@ main() {
   fi
 
   if [[ "$OSTYPE" == "linux"* ]]; then
-    if [[ $lhelper == true ]]; then
-      sudo apt-get install -qq ninja-build
-    else
-      sudo apt-get install -qq libfuse2 ninja-build wayland-protocols libsdl2-dev libfreetype6
-    fi
+    sudo apt-get install -qq libfuse2 ninja-build wayland-protocols libsdl2-dev libfreetype6
```

Comment:
Do we stil need `libsdl2-dev` and `libfreetype6`? Aren't those handled by our fallbacks?

Reply:
IMO unrelated to this PR.

In my opinion this file should be removed completely in favor of docs mentioning how to install system dependencies

Reply:
Yeah I agree.
Also considering that for example `libfuse2` is only useful when building the appimage, so people might not even want to install it.